src/chains/supportedChains.evm.ts

```diff
@@ -1226,4 +1226,28 @@ export const supportedEVMChains: EVMChain[] = [
       rpcUrls: ['https://mainnet.unichain.org'],
     },
   },
+
+  {
+    key: ChainKey.INK,
+    chainType: ChainType.EVM,
+    name: 'Ink',
+    coin: CoinKey.ETH,
+    id: 57073,
+    mainnet: true,
+    logoURI:
+      'https://raw.githubusercontent.com/lifinance/types/main/src/assets/icons/chains/ink.svg',
+    multicallAddress: multicallAddresses[ChainId.INK],
+
```

Comment:
Can we please add this flag right away, because Ink is already supported in our relayer service. Requires pull from main with updated types. 🙏

```suggestion
relayerSupported: true,
```

Reply:
Of course 👌